modules/viewing-registered-model-versions.adoc

```diff
@@ -0,0 +1,46 @@
+:_module-type: PROCEDURE
+
+[id="viewing-registered-model-versions_{context}"]
+= Viewing registered model versions
+
+[role='_abstract']
+You can view the details of model versions that are registered in {productname-short}, such as the version metadata and deployment information.
+
+.Prerequisites
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {oai-user-group} or {oai-admin-group}) in OpenShift.
+endif::[]
+ifdef::upstream[]
+* If you are using {productname-short} groups, you are part of the user group or admin group (for example, {odh-user-group} or {odh-admin-group}) in OpenShift.
+endif::[]
+* An available model registry exists in your deployment, and contains at least 1 registered model.
+* You have access to the model registry that contains the model version that you want to view.
+
+.Procedure
+. From the {productname-short} dashboard, click *Model Registry*.
+. From the *Model registry* dropdown menu, select the model registry that contains the model version that you want to view.
+. Click the name of a model to view its versions.
++
+The details page for the model opens.
+. On the *Versions* tab, you can view registered versions of the model.
+. Click the name of a version to view more details.
++
+The details page for the version opens.
+. On the *Details* tab, you can view the description, labels, custom properties, version ID, author, and *Last modified* and *Registered* timestamps for the model. You can also view the source model format, and location information for the model.
```

Comment:
- Minor suggestion to either bold all UI field names or none (Last modified and Registered stand out). 
- Remove comma between "format, and" (You can also view the source model format and location information for the model.)

Reply:
Agreed - updated to not bold any, and adjusted the wording for the two that were in bold to read more clearly.